litmus-portal/frontend/src/views/Analytics/ApplicationDashboards/Steps/TuneTheQueries/EditPanelsWizard/index.tsx

```diff
@@ -250,18 +250,7 @@ const EditPanelsWizard: React.FC<EditPanelsWizardProps> = ({
     return newPanel;
   };
 
-  const handleCreatePanel = () => {
-    const existingPanels: PanelDetails[] =
-      dashboardDetails.selectedPanels ?? [];
-    const newPanel = getNewPanel();
-    existingPanels.push(newPanel);
-    setDashboardDetails({
-      ...dashboardDetails,
-      selectedPanels: existingPanels,
-    });
-    generatePanelGroupsList(existingPanels);
-    setUpdate(true);
-  };
```

Comment:
why have we removed this function @whitetiger1399

Reply:
This required, please revert this change.

Reply:
It got deleted on autoSave. Don't know why it happened. But it has happened twice. Will revert it